CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ### Features
 
+* `[jest-config]` According to ([#5828](https://github.com/facebook/jest/issues/5828)), moves `deprecationEntries` from
+  `jest-config` to `jest-validate`.
+  ([#6067](https://github.com/facebook/jest/pull/6067))
+* `[jest-validate]` According to ([#5828](https://github.com/facebook/jest/issues/5828)), moves `deprecationEntries` from
+  `jest-config` to `jest-validate`.
+  ([#6067](https://github.com/facebook/jest/pull/6067))
+* `[jest-runtime]` Fix typo.
```

Comment:
if this is included, it should be under `chores`. but  don't think a typo fix needs to be in the changelog, especially as it's not user visible (the typo wasn't in a message show to the user.)